mxcube3/routes/SampleCentring.py

```diff
@@ -594,7 +601,7 @@ def accept_centring():
     """
     Accept the centring position.
     """
-    mxcube.diffractometer.acceptCentring()
+    mxcube.diffractometer.saveCurrentPos() #acceptCentring()
```

Comment:
this does not exists in the generic difractometer

Reply:
Aha, okey I will revert back to acceptCentring then.

Thanks